snap/snapcraft.yaml

```diff
@@ -325,123 +325,17 @@ parts:
-        install I2C/eetool.sh ${SNAPCRAFT_PART_INSTALL}/wigwag/system/bin/eetool
-        install -d ${SNAPCRAFT_PART_INSTALL}/etc/network
-        install -d ${SNAPCRAFT_PART_INSTALL}/etc/udev
-        install -d ${SNAPCRAFT_PART_INSTALL}/etc/udev/rules.d
-        install -d ${SNAPCRAFT_PART_INSTALL}/wigwag/support
-        install initscripts/UDEV/96-local.rules ${SNAPCRAFT_PART_INSTALL}/etc/udev/rules.d/96-local.rules
-      organize:
-        conf: wigwag/wwrelay-utils/conf
-        6BSMD: wigwag/wwrelay-utils/6BSMD
-        common: wigwag/wwrelay-utils/common
-        slip-radio: wigwag/wwrelay-utils/slip-radio
-        .b: wigwag/wwrelay-utils/.b
-        initscripts: wigwag/wwrelay-utils/initscripts
-        GPIO: wigwag/wwrelay-utils/GPIO
-        I2C: wigwag/wwrelay-utils/I2C
-        debug_scripts: wigwag/wwrelay-utils/debug_scripts
-        dev-tools: wigwag/wwrelay-utils/dev-tools
-        I2C: wigwag/wwrelay-utils/I2C
-      filesets:
-        etc:
-          - etc/dnsmasq.conf
-          - etc/dnsmasq.d/*
-          - etc/modprobe.d/*
-          - etc/profile.d/wigwagpath.sh
-          - etc/init.d/devjssupport
-          - etc/init.d/relayterm
-          - etc/init.d/wwfunctions
-          - etc/init.d/wwrelay
-          - etc/network
-          - etc/udev
-        conf:
-          - wigwag/wwrelay-utils/conf/*
-        versions:
-          - wigwag/etc/versions.json
-        utils:
-          - wigwag/devicejs-core-modules/rsmi/bin/slipcomms-arm
-          - wigwag/devicejs-core-modules/rsmi/bin/cc2530prog-arm
-          - wigwag/wwrelay-utils/6BSMD/*
-          - wigwag/wwrelay-utils/common/*
-          - wigwag/wwrelay-utils/slip-radio/*
-          - wigwag/wwrelay-utils/.b
-          - wigwag/wwrelay-utils/initscripts/*
-          - wigwag/wwrelay-utils/GPIO/*
-          - wigwag/wwrelay-utils/I2C/*
-          - -wigwag/wwrelay-utils/GPIO/led.sh
-          - -wigwag/wwrelay-utils/GPIO/pinctrl.sh
-          - -wigwag/wwrelay-utils/GPIO/leddaemon
-          - -wigwag/wwrelay-utils/I2C/eetool.sh
-        debug:
-          - wigwag/wwrelay-utils/debug_scripts
-        devtools:
-          - wigwag/wwrelay-utils/dev-tools
-          - -wigwag/wwrelay-utils/dev-tools/bin/stopwatchdog.sh
-          - -wigwag/wwrelay-utils/dev-tools/bin/info.sh
-        system:
-          - wigwag/system/bin/ccommon.sh
-          - wigwag/system/bin/stopwatchdog
-          - wigwag/system/bin/restartjob.sh
-          - wigwag/system/bin/led
-          - wigwag/system/bin/pinctrl
-          - wigwag/system/bin/leddaemon
-          - wigwag/system/bin/dogProgrammer
-          - wigwag/system/bin/eetool
-        support:
-          - wigwag/support
-      stage:
-        - $etc
-        - $conf
-        - $versions
-        - $utils
-        - $debug
-        - $devtools
-        - $system
-        - $support
+        install ${SNAPCRAFT_PROJECT_DIR}/files/wwrelay-utils/version.json ${SNAPCRAFT_PART_INSTALL}/wigwag/etc/
```

Comment:
since the part was renamed to pe-utils, we should move these files to a pe-utils folder.

Reply:
Good catch - will fix that right now